lib/blurb/ad_group.rb

```diff
@@ -0,0 +1,84 @@
+module Blurb
+  class AdGroup < BaseResource
+    SPONSORED_PRODUCTS = "sp"
+    SPONSORED_BRANDS = "hsa"
```

Comment:
You can add these constants to `BaseResource` since they are duplicated in a handful of the resource classes.